README.md

```diff
@@ -24,7 +24,8 @@ const combinedComposables = composable0.compose(composable1, composable2, compos
 
 ### Descriptor
 
-**Composable descriptor** (or just **Descriptor**) is a meta data object. It's a collection of properties that contain the information necessary to create an object instance by a *composable*.
+**Composable descriptor** (or just **Descriptor**) is a meta data object. It's a collection of properties that contain
+the information necessary to create an object instance by a *composable*.
```

Comment:
Don't fix a problem with *your editor config* by breaking the README for *everybody except you.*